components/card/examples/01-card-examples.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import {
+  Card,
+  CardBody,
+  CardFooter,
+  CardHeader,
+} from '../src/index';
+
+export default () => (
+  <div>
+    <Card hasShadow={false}>
```

Comment:
This should probably be inverted so you can say `<Card hasShadow />`. This would mean the default should be it does not have a shadow or we should change the `hasShadow` to `noShadow` or something similar.

Reply:
If you do `hasShadow` or `hasShadow={true}` is different syntax to the same thing. The portion I was trying to point out was the default value being a false value versus defaulting to a true value. In the way currently, we would never need `hasShadow={true}` because it defaults to true. That is the part I think we should revise.